src/Tizen.Security/Tizen.Security/Privilege.cs

```diff
@@ -153,6 +153,7 @@ public static string GetDescription(string apiVersion, string privilege, Package
         /// <exception cref="System.OutOfMemoryException">Thrown when out of memory occurs.</exception>
         /// <exception cref="System.InvalidOperationException">Thrown when an internal error occurs.</exception>
         /// <exception cref="System.NotSupportedException">The required feature is not supported.</exception>
+        [Obsolete("Deprecated since API level 11.")]
```

Comment:
it's better to include information about replacement, is there any a way a user can achieve similar result with other API?

Reply:
There's no replacement of it. Would it be better to add 'Please don't use it' message?

Reply:
There' no replacement of it. Would it be better to add 'Please don't use it' message instead?

Reply:
@wiertel Please review this PR.